moonbeam-api-augment/.prettierrc.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I will keep only rules that are necessary. I need to use `prettier-plugin-jsdoc`. By default, prettier is not formatting comments. But when we do check in the pipeline `check-editorconfig` it fails because comments are too long. So I added the plugin to format comments so they are matching editor config rules.